test/e2e/e2e.go

```diff
@@ -32,11 +32,16 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/v1"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
```

Comment:
> Is this what we're doing now? Seems weird that we're hard coding version into clientset?

It's how the external client is versioned at the moment.  We are transitioning to use the versioned client so that we can eventually make use of the `kubernetes/client-go` client.

Using the internal client is now discouraged in non-api server locations.